ajax/libs/lawnchair/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "lawnchair",
+  "version": "0.6.4",
+  "license": "MIT",
+  "filename": "Lawnchair.min.js",
+  "keywords": [
+    "lawnchair",
+    "Gruntfile",
+    "lightweight",
+    "json",
+    "document"
+  ],
+  "description": "simple json storage",
+  "author": "Brian Leroux",
+  "homepage": "http://brian.io/lawnchair/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/brianleroux/lawnchair.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/brianleroux/lawnchair.git",
+    "fileMap": [
+      {
+        "basePath": "src",
+        "files": [
+          "**/*"
```

Comment:
It's not a good idea to use `**/*` under src, I think we discussed about this for several times.